mgmtsystem/__manifest__.py

```diff
@@ -39,5 +23,5 @@
         'views/res_config.xml'
     ],
     "demo": [],
-    'installable': False,
+    'installable': True,
 }
```

Comment:
@eugen-don @max3903 @dreispt related, I notice there is a dependency on `board` that generates warnings during installation, and does not seem to be used inside this `mgtsystem` module.

The warnings/errors I get are 

``
WARNING test-mgtsystem-v10c odoo.modules.registry: Models have no table: board.board.
INFO test-mgtsystem-v10c odoo.modules.registry: Recreate table of model board.board.
ERROR test-mgtsystem-v10c odoo.modules.registry: Model board.board has no table.
``